src/tests/class_tests/openms/source/IsoSpec_test.cpp

```diff
@@ -521,7 +514,7 @@ START_SECTION(( void IsoSpecTotalProbWrapper::run() ))
 
   // human insulin
   IsotopeDistribution iso_result3 = IsoSpecTotalProbWrapper(EmpiricalFormula("C520H817N139O147S8"), total_prob, do_trim).run();
-  TEST_EQUAL(iso_result3.size(), 19616);
+  TEST_EQUAL(iso_result3.size(), 19615);
```

Comment:
now it's correct?

Reply:
Actually, in a sense both values are sorta correct. We compute the probabilities of configurations a bit differently now, compared to what used to be. Mathematically it's equivalent, but we do different rounding errors during calculations now. In this case it resulted in one less configuration being needed to cross the requested probability threshold. In either case the rounding errors are much less than the tolerances on element abundances given by IUPAC, so it shouldn't be important.